packages/build/src/test/js-transform_test.ts

```diff
@@ -336,4 +336,21 @@ suite('jsTransform', () => {
         result,
         `console.log(new Promise((res, rej) => _require.default(['./bar.js'], res, rej)));`);
   });
+
+  // https://github.com/babel/babel/issues/7506
+  test('does not include empty native constructor wrappers', () => {
+    const input = stripIndent(`
+      class TestElement extends HTMLElement {
+        constructor() {
+          super();
+          this.x = 1234;
+        }
+      }
+
+      window.customElements.define("test-element", TestElement);
+    `);
+    const result = jsTransform(input, {compileToEs5: true});
+
+    assert.notInclude(result, "function Wrapper() {}");
```

Comment:
What about checking for `wrapNativeSuper`, since it's the name of the helper function that contains this line? Maybe less brittle.

Reply:
I think just wrapNativeSuper is fine. If you did do both, make sure it's an *or*, since an *and* would be _more_ brittle.

Reply:
Ok, I'll just do `wrapNativeSuper` then.